tests/export.bats

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "extract" {
+  touch ${TESTDIR}/reference-time-file
+  for source in scratch alpine; do
+    cid=$(buildah from --pull=true --signature-policy ${TESTSDIR}/policy.json ${source})
+    mnt=$(buildah mount $cid)
+    touch ${mnt}/export.file
+    buildah umount $cid
+    buildah export "$cid" > ${TESTDIR}/${source}.tar
+    buildah export "$cid" > ${TESTDIR}/${source}.tar
```

Comment:
It's fine, but did you mean to do the same export twice?  Are you just testing the overwrite of the tar?